calculation/gmhazard_calc/gmhazard_calc/dbs/IMDB.py

```diff
@@ -330,7 +330,7 @@ def rupture_names(self) -> Union[None, np.ndarray]:
 
     @check_open
     def im_data(
-        self, station: str, im: Optional[Union[List[IM], IM]] = None, inter_intra: bool = False,
+        self, station: str, im: Optional[Union[List[IM], IM]] = None, incl_inter_intra: bool = False,
```

Comment:
I recon `incl_with_between_sigma` or `incl_rand_effects_sigma` might be better? Not a fan of `inter` and `intra`